src/Microsoft.DotNet.VersionTools/Util/FileUtils.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.DotNet.VersionTools.Util
+{
+    static class FileUtils
```

Comment:
I omitted the access modifier to follow the pattern used by the other classes in this folder. @dagood do you know why this pattern was established?  Do you have a preference on whether explicit access modifiers should be used here?